packages/flutter/lib/src/cupertino/date_picker.dart

```diff
@@ -1053,6 +1139,29 @@ class _CupertinoTimerPickerState extends State<CupertinoTimerPicker> {
     );
   }
 
+  /// Sets the picker to be the initial time.
+  void _desiredDurationChanged() {
+    final Duration desiredTimerDuration = widget.controller.desiredTimerDuration;
+    final Duration animationDuration = widget.controller.resetAnimationDuration;
+
+    assert(desiredTimerDuration >= Duration.zero);
```

Comment:
Should we add explanations for assertions?

Reply:
I feel like they're pretty self explanatory?

Reply:
The boolean conditions may be self-explanatory, but what I mean is that it might be nice to suggest a developer why it may have failed, if we're able to suggest any such information.